dbms/src/Storages/KVStore/MultiRaft/Disagg/CheckpointIngestInfo.cpp

```diff
@@ -193,10 +242,20 @@ bool CheckpointIngestInfo::forciblyClean(TMTContext & tmt, UInt64 region_id, boo
     Page page = uni_ps->read(page_id, nullptr, snapshot, /*throw_on_not_exist*/ false);
     if (unlikely(page.isValid()))
     {
-        CheckpointIngestInfo::removeFromLocal(tmt, region_id);
+        removeFromLocal(tmt, region_id);
         return true;
     }
     return false;
 }
 
+bool CheckpointIngestInfo::forciblyClean(
+    TMTContext & tmt,
+    const TiFlashRaftProxyHelper * proxy_helper,
+    UInt64 region_id)
+{
+    UNUSED(proxy_helper);
```

Comment:
This part will be reconsidered in another PR. The commented lines are to be removed.